TODO/introducing-pokedex-org.md

```diff
@@ -1,184 +1,171 @@
+我决定将这些想法放在一起构建一个像原生应用一样的丰富交互体验的 `webapp`，但其实它仅仅只是一个网站。根据 Chrome 小组的准则，我构建了 [Pokedex.org](http://pokedex.org) - 一个离线工作的[渐进网页应用](https://infrequently.org/2015/06/progressive-apps-escaping-tabs-without-losing-our-soul/)，它可以从主屏幕启动，甚至在普通的 `Android` 手机上运行在 60 `FPS`。这篇博客文章将解释我是如何做到的。
 
-<video width="400" poster="//nolanlawson.s3.amazonaws.com/vid/DeliriousNeedyAnophelesmosquito.png"><source src="http://nolanlawson.s3.amazonaws.com/vid/DeliriousNeedyAnophelesmosquito.webm" type="video/webm"> <source src="http://nolanlawson.s3.amazonaws.com/vid/DeliriousNeedyAnophelesmosquito.mp4" type="video/mp4"></video> 
+## 宠物小精灵 - 一个雄心勃勃的目标
 
-The first issue is getting the data, which is easy thanks to the wonderful [Pokéapi](http://pokeapi.co/). The second issue is that, if we want the app to work offline, the database is far too large to keep in memory, so we'll need some clever use of IndexedDB and/or ServiceWorker.
+对于那些未初始化到宠物小精灵的世界里，一个图鉴包含数以百计的娇媚的小生物，以及他们的属性，类型，进化和移动信息的百科全书。对于一个儿童游戏来说恐怕数据量的巨大是令人惊讶的(假如你希望烧脑，可以通过仔细研究[努力值](http://bulbapedia.bulbagarden.net/wiki/Effort_values))。所以这是一个雄心勃勃的Web应用程序的理想目标。
 
-For this app, I decided to use [PouchDB](http://pouchdb.com/) for the Pokémon data (because it's good at sync), as well as [LocalForage](https://github.com/mozilla/localForage) for app state data (because it has a nice key-value API). Both PouchDB and LocalForage are using IndexedDB inside a web worker, which means any database operations are [fully non-blocking](http://nolanlawson.com/2015/09/29/indexeddb-websql-localstorage-what-blocks-the-dom/).
+<video width="400" poster="//nolanlawson.s3.amazonaws.com/vid/DeliriousNeedyAnophelesmosquito.png"><source src="http://nolanlawson.s3.amazonaws.com/vid/DeliriousNeedyAnophelesmosquito.webm" type="video/webm">	<source src="http://nolanlawson.s3.amazonaws.com/vid/DeliriousNeedyAnophelesmosquito.mp4" type="video/mp4"></video>	
 
-However, it's also true that the Pokémon data isn't immediately available when the site is first loaded, because it takes awhile to sync from the server. So I'm also using a fallback strategy of "local first, then remote":
+第一个问题是获取数据，这个很容易，多亏了精彩的[Pokéapi](http://pokeapi.co/)。第二个问题是，如果我们希望应用程序脱机工作，数据库过于庞大，不能保持在内存中，所以我们需要一些聪明的办法来使用 `IndexedDB` 和/或 `ServiceWorker`。
+
+这个程序，我决定使用[PouchDB]（http://pouchdb.com/）保存宠物小精灵数据（因为它的同步良好），同时使用[LocalForage](https://github.com/mozilla/localForage)作为应用的状态数据存储（因为它有一个很好的键值API）。无论 `PouchDB` 和 `LocalForage` 使用 `IndexedDB` 的 `Web Worder`，这意味着任何数据库操作者将是[完全无阻塞](http://nolanlawson.com/2015/09/29/indexeddb-websql-localstorage-what-blocks-the-dom/)。
```

Comment:
because it's good at sync => 因为它擅长同步
key-value API 建议不译
Both PouchDB and LocalForage are using IndexedDB inside a web worker =>  PouchDB 和 LocalForage 都在 web worker 中使用 IndexedDB